router/voteCallbacks.go

```diff
@@ -31,7 +31,9 @@ func (r *Router) submitRawTx(request routerRequest) {
 	log.Infof("broadcasting tx hash:%s", res.Hash)
 	var response types.MetaResponse
 	response.Payload = fmt.Sprintf("%x", res.Data) // return nullifier or other info
-	request.Send(r.buildReply(request, &response))
+	if err = request.Send(r.buildReply(request, &response)); err != nil {
```

Comment:
`err :=`

Reply:
These `=` were introduced in purpose, I don't really see the need for declaring a new variable on this specific scope. I do see it makes sense in some others however.